stable.js

```diff
@@ -1099,6 +1099,25 @@ var idrinth = {
         },
         controls: null,
         tooltipTO: null,
+        buildModal:function(title,content,altFunc) {
```

Comment:
The issue is, that I'd need to typecheck both content itself as well as any children it might have.
content can be used as a string or the usual configuration-object, throwing anything else out, that doesn't see to fit is imo the best in that case.
I fear checking the whole content-tree might just be a bit too much overhead.